docs/tutorials/articles/sales_dashboard/step_02/step_02.md

```diff
@@ -0,0 +1,198 @@
+---
+hide:
+  - toc
+---
+
+The full code for this step is available 
+[here](https://github.com/AlexandreSajus/taipy-course/tree/main/3_styling)
+
+This step will be about styling the application. We will add more filters, layout the visual element and 
+regroup them in parts.
+
+![Styling Application](images/styling_app.png){ width=90% : .tp-image-border }
+
+We can still use the same code as the previous step, but let's recreate the page from scratch:
+
+```python
+with tgb.Page() as page:
+    with tgb.part(class_name="container"):
+        tgb.text("# Sales by **State**", mode="md")
+```
+
+[Part](../../../../refmans/gui/viselements/generic/part.md) allows you to group and style visual elements together.
+Here, the [container](../../../../userman/gui/styling/stylekit.md) class is a predefined style
+that will limit the width of visual elements contained in the part.
+
+[Text](../../../../refmans/gui/viselements/generic/text.md) is a simple text visual element. Here we use the
+Markdown (md) mode to display a title and make the word "State". We can also color bold text in orange
+by using CSS. Create a new CSS file with the same name as the Python file and add the following code:
+
+```css
+strong,
+b {
+    font-weight: bold;
+    color: var(--color-primary);
+}
+```
+
+Let's now add a new container for the filters:
+
+```python
+        with tgb.part(class_name="card"):
```

Comment:
Why a part if there's only a block?
Why all those parts all over the place??

Reply:
Parts are a good way to visually separate in the code the contents of different parts of the app